Maquetado/Setup Proyecto/.gitignore

```diff
@@ -0,0 +1,10 @@
+### Sass ###
```

Comment:
I think that **Setup Proyecto** folder is unnecessary. Please put the files directly inside of Maquetado folder.